packages/formation-react/stories/address-block.stories.mdx

```diff
@@ -0,0 +1,14 @@
+import { Meta, Story, Canvas } from '@storybook/addon-docs/blocks';
+
+<Meta title="Library/Address block" />
+
+<Canvas>
+  <Story name="Default">
+    <p class="va-address-block">
+      Department of Veterans Affairs Claims Intake Center <br/>
+      Attention: C-123 Claims<br/>
+      PO Box 5088<br/>
+      Janesville, WI 53547-5088<br/>
+    </p>
+  </Story>
+</Canvas>
```

Comment:
Removing the `<Story>` but leaving `<Canvas>` seems like it might be ideal:

![image](https://user-images.githubusercontent.com/2008881/100292098-6a1ca880-2f34-11eb-8696-eb03fb2617e8.png)

Reply:
Ahhh, yeah here we go. This is what I wanted. I don't so much care about having the `Show code` button, but I wanted the clearer distinction between the component we're demonstrating and the surrounding text.